### PR TITLE
chore: upgrade deps, fix broken event links

### PR DESCRIPTION
**Pull request by harrysolovay**

### CODE REVIEW

File: src/components/Link.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import {Link} from 'gatsby';
 import ExternalLink from './ExternalLink';
 
 export default props => {
-  const {href, to, ...rest} = props;
+  let {href, to, ...rest} = props;
```
Comment (sammartinez) [inline src/components/Link.js:5]: Were these changes needed?

Comment (harrysolovay) [reply src/components/Link.js:5]: The security patch was related to an outdated version of some websocket package, which was a dependency of Gatsby's. I needed to updated Gatsby. Upon doing so, the internal link behavior broke. A link `event/[...rest]` no longer resolved to `https://community.amplify.aws/event/[...rest]`. Instead it resolved relative to the current page (aka, `https://community.amplify.aws/event/event/[...rest]`). I needed to safeguard against this behavior.

**Comment by sammartinez** [comment]
Can we verify the [build error](https://app.circleci.com/pipelines/github/aws-amplify/community/647/workflows/d0b0c3c6-87f0-49d5-82af-c8dac4457cbd/jobs/648/steps)? After this I can approve it

**Comment by harrysolovay** [comment]
@sammartinez we not use GitHub actions to CI testing. Please ignore the CircleCI check failure. We should remove that. In fact, give me the green light, and I'll do so right now.
